website/docs/r/guardduty_member.html.markdown

```diff
@@ -39,12 +41,25 @@ The following arguments are supported:
 * `account_id` - (Required) AWS account ID for member account.
 * `detector_id` - (Required) The detector ID of the GuardDuty account where you want to create member accounts.
 * `email` - (Required) Email address for member account.
+* `invite` - (Optional) Specifies whether to invite the account to GuardDuty as members.
+* `invitation_message` - (Optional) Message for invitation.
+* `disable_email_notification` - (Optional) Specifies whether an email notification is sent to the accounts.
+
+## Timeouts
+
+`aws_guardduty_member` provides the following [Timeouts](/docs/configuration/resources.html#timeouts)
+configuration options:
+
+- `create` - (Default `60s`) How long to wait for a verification to be done against inviting GuardDuty member account.
+
 
 ## Attributes Reference
 
 The following additional attributes are exported:
 
 * `id` - The ID of the GuardDuty member
+* `invited` - If you already have invited another account as a member, then this value is calculated based on `relationship_status`. This is marked as true if the status is either "Disabled", "Enabled", "Invited", or "EmailVerificationInProgress".
```

Comment:
Upon switching the `d.Set("invited", ...)` to `d.Set("invite", ...)` as noted above, I will move this documentation to the `invite` argument. 👍